services/app-api/handlers/reports/create.ts

```diff
@@ -44,15 +51,17 @@ export const createReport = handler(async (event, _context) => {
       if (validatedFieldData) {
         // post validated field data to s3 bucket
         const fieldDataParams: S3Put = {
-          Bucket: process.env.MCPAR_FORM_BUCKET!,
+          Bucket:
+            reportFormBuckets[reportType as keyof typeof reportFormBuckets],
```

Comment:
Suggestion: For all the places across `create`, `archive`, `update`, and `fetch` where we're setting the `TableName:` or `Bucket:` param multiple times, we could get the report or bucket once at the top of the file, set it as a const, then reference it below instead of accessing that constant multiple times. Just makes it a bit cleaner.

_Applies to all similar instances below_